### ISSUE TITLE
Feat: Notification wall filter (just my mentions/all timeline)

### ISSUE THREAD

**Issue by gbonaspetti** [tracim/tracim#3324]
## Feature description

It should be possible to add/remove the filter "just my mentions".

- [ ] by default it's displayed just "my mentions"
- [ ] add filter button
- [ ] on click shows/hide notifications

## Prerequisites

#2840 

## Impacted components

frontend

## Interface

**Comment by Skylsmoi** [comment]
I'm wondering if the switch of icon position wont be a bit unaesthetic.
I like the idea of having a + and a x for add and remove though
Random idea : maybe having it at the same place and just rotate it on click ?

**Comment by gbonaspetti** [comment]
[@celiamgtt] @Skylsmoi after discussion, we decided to make an intermediate design while we only have one filter, but for the evolution of the design I agree with you, even make + and - instead of + and x.

**Comment by raphj** [comment]
If I understand correctly, the the "filters menu" referred to in this issue is this list of filters, right? @gbonaspetti 
![image](https://user-images.githubusercontent.com/3817365/87907755-76b06200-ca65-11ea-8d3a-fdf61919c0f7.png)

**Comment by gbonaspetti** [comment]
> If I understand correctly, the the "filters menu" referred to in this issue is this list of filters, right? @gbonaspetti
> ![image](https://user-images.githubusercontent.com/3817365/87907755-76b06200-ca65-11ea-8d3a-fdf61919c0f7.png)

Yes, there is the slider button ![notifs_mur_tag_header](https://user-images.githubusercontent.com/35422692/87914452-ef68eb80-ca70-11ea-9f84-546e126b50b7.png) that opens this menu. But the design of the menu itself (your image) will be changed will be changed for an intermediate one.

**Comment by grignards** [comment]
As an example, here is my usage, tuned for:
- getting my mentions
- not being disturbing

- I've hidden the notifications button  (and mentions red bubbles) in the sidebar as I find them too disturbing visually (not related but I also hide the space types as "confidential" spaces are in red which I find really uncomfortable).
- So I'm not using the notification wall at all: its signal-to-noise ratio is far too low for my usage.
- I configured my email client to filter out notification mails and only keep those containing a mention for me/all => this allows to get quick notification of mentions
- I come at chosen times to read the last activities page to keep myself informed of changes/news.
- BTW I also hide the activities "member join/leave" as I'm not interested into them.

**Comment by josiealgoo** [comment]
I have updated the content of the ticket!
